tests/test_sensors/test_led.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+from src.sensors.led import FixedLED
+
+@pytest.fixture
```

Comment:
Could this fixture be in a conftest.py file in the base `tests` directory?

Reply:
Link is broken for me, for those looking in I found [this](https://stackoverflow.com/questions/33654052/how-to-share-object-from-fixture-to-all-tests-using-pytest) to be useful, thanks for the tip though :)